nilearn/decoding/decoder.py

```diff
@@ -1450,6 +1479,9 @@ def __sklearn_tags__(self):
         tags.estimator_type = "regressor"
         tags.regressor_tags = RegressorTags()
 
+        # TODO
+        # check sklearn RegressorMixin and MultiOutputMixin to adapt this
```

Comment:
and that

Reply:
same reasoning for this

Reply:
See https://github.com/nilearn/nilearn/issues/5583